examples/03-widgets/clip-volume.py

```diff
@@ -0,0 +1,45 @@
+"""
+.. _clip_volume_widget_example:
+
+Clip Volume Widget
+~~~~~~~~~~~~~~~~~~
+If you have a :class:`pyvista.UniformGrid`, you can clip it using the
+:func:`pyvista.Plotter.add_volume_clipper` widget.
+
+
+.. image:: ../../images/gifs/box-clip.gif
```

Comment:
Is this gif (`add_mesh_clip_box` on PolyData Nefertiti) relevant for this example?

Reply:
It's not. Sorry for the weak PR. It should have been labeled "draft".